.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+Contributing Guidelines
+--------------------------------------------------
+
+This document provides general guidelines about how to contribute to the project. Keep in mind these important things before you start contributing.
+
+### Reporting issues
+
+* Use [github issues](https://github.com/IDme/ID.me-WebVerify-SDK-Android/issues) to report a bug.
+* Before creating a new issue:
+  * Make sure you are using the [latest release](https://github.com/IDme/ID.me-WebVerify-SDK-Android/releases).
+  * Check if the issue was [already reported or fixed](https://github.com/IDme/ID.me-WebVerify-SDK-Android/issues?utf8=%E2%9C%93&q=is%3Aissue). Notice that it may not be released yet.
+  * If you found a match add a brief comment "I have the same problem" or "+1". This helps prioritize the issues addressing the most common and critical ones first. If possible, add additional information to help us reproduce and fix the issue. Please use your best judgement.    
```

Comment:
I strongly disagree with what's stated in the first sentence. I'm sick of receiving "+1" emails and I believe other issues' subscribers too. For this GH have created the reactions to comments, issues and PRs. I prefer to encourage them instead.